packages/@o3r/design/testing/mocks/design-token-theme.json

```
@@ -16,6 +16,18 @@
         "o3rPrivate": true
       }
     },
+    "var-ref-to-var1-private": {
+      "$value": "{example.var1-private}"
+    },
+    "var-private-ref": {
+      "$value": "{example.var1}",
+      "$extensions": {
+        "o3rPrivate": true
+      }
+    },
+    "var-private-ref-to-public-var ": {
```

```suggestion
    "var-private-ref-to-public-var": {
```
(picky) also update this name in the spec file